.expeditor/license_scout.sh

```diff
@@ -5,6 +5,10 @@ set -eou pipefail
 if [[ "${EXPEDITOR:-false}" == "true" ]]; then
   apt-get update
   apt-get install -y libpq-dev libsqlite3-dev
+  # Pin ruby to 2.5.3 since chef-server tests heavily depend
+  asdf global ruby 2.5.3
```

Comment:
Any reason not to use `local` instead of `global`?

Reply:
local does not update the default tools_version and current license_scout gem (2.1.5) does not support ruby 2.6.1